IceCubesApp/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -82,7 +82,7 @@
 "settings.display.theme.systemColor" = "Systemeinstellung verwenden";
 "settings.general.browser" = "Browser";
 "settings.general.browser.in-app" = "In-App-Browser";
-"settings.general.browser.in-app.readerview" = "In-App Browser Reader View";
+"settings.general.browser.in-app.readerview" = "In-App-Browser-Lesemodus";
```

Comment:
I believe that in the German localization of Apple software the Safari Reader is called "Reader" in German too. 

https://applelocalization.com/?q=reader&l=English&l=German

Reply:
It is, but other browsers use “Lesemodus” or similar terms. Personally, I’d always prefer actual translations instead of just declaring English words “fixed terms”..?

Reply:
Me too! Apparently, Apple regard the term “Safari Reader” so quintessential and untranslatable (for marketing reasons?). 

My opinion? Is Safari the in-app browser, then it should be "Reader" for consistency-sake. If the in-app browser is a white-label browser offering a reader mode, then Lesemodus is the better choice.

I'll leave the final decision with you, of course. This is just my two cents 🙂

Reply:
The in-app browser is Ice Cubes. ;-)